src/components/TrendLine/TrendLine.js

```diff
@@ -0,0 +1,186 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { omit } from './utils/index';
+import {
+  buildLinearPath,
+  buildSmoothPath,
+  injectStyleTag,
+} from './helpers/DOM.helpers';
+import { normalize } from './helpers/math.helpers';
+import { generateId } from './helpers/misc.helpers';
+import { generateAutoDrawCss, normalizeDataset } from './Trend.helpers';
+
+const propTypes = {
+  data: PropTypes.arrayOf(
+    PropTypes.oneOfType([
+      PropTypes.number,
+      PropTypes.shape({
+        value: PropTypes.number,
+      }),
+    ]).isRequired
+  ).isRequired,
+  smooth: PropTypes.bool,
+  autoDraw: PropTypes.bool,
+  autoDrawDuration: PropTypes.number,
+  autoDrawEasing: PropTypes.oneOfType(['ease-in', 'ease-out', 'linear']),
+  /** @uxpinignoreprop */
+  width: PropTypes.number,
+  /** @uxpinignoreprop */
+  height: PropTypes.number,
+  padding: PropTypes.number,
+  radius: PropTypes.number,
+  gradient: PropTypes.arrayOf(PropTypes.string),
+  strokeWidth: PropTypes.number,
+};
+
+const defaultProps = {
+  smooth: true,
+  radius: 10,
+  padding: 8,
+  strokeWidth: 3,
+  autoDraw: true,
+  autoDrawDuration: 1600,
+  autoDrawEasing: 'ease-out',
+  // eslint-disable-next-line react/default-props-match-prop-types
+  data: [0, 2, 5, 9, 5, 10, 3, 5, 0, 0, 1, 8, 2, 9, 0],
+  gradient: ['#00c6ff', '#F0F', '#FF0'],
+};
+
+class TrendLine extends Component {
```

Comment:
It seems that `TrendLine` don't have a preset

Reply:
Preset added.